books/RayTracingTheRestOfYourLife.html

```diff
@@ -1819,7 +1819,7 @@
         bool scatter(const ray& r_in, const hit_record& rec, color& attenuation, ray& scattered)
         const override {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-            auto scatter_direction = random_in_hemisphere(rec.normal);
+            auto scatter_direction = random_on_hemisphere(rec.normal);
```

Comment:
I _think_ that *on* is really what we want. If we really do want *on*, then we'll need to implement it. Also note that this code is quickly replaced later in the text.